test/csuite/wt7989_compact_checkpoint/main.c

```diff
@@ -231,15 +222,17 @@ thread_func_compact(void *arg)
 {
     struct thread_data *td;
     WT_SESSION *session;
+    // uint64_t ready_counter_local;
```

Comment:
Please remove this commented-out variable declaration.

Reply:
Fixed.